### PR TITLE
feat: remove tabulate dependency

### PR DESCRIPTION
**Pull request by relikd**
fixes: #20

### CODE REVIEW

File: remotezip.py [hunk 1]
```diff
@@ -259,12 +258,26 @@ def _list_files(url, support_suffix_range, filenames):
...
-        print(tabulate(data, headers='firstrow'))
+        _printTable(data, ('Length', 'DateTime', 'Name'), '><<')
+
+
+def _printTable(data, header, align):
+    # get max col width & prepare formatting string
+    col_w = [len(col) for col in header]
+    for row in data:
+        col_w = [max(w, len(str(x))) for w, x in zip(col_w, row)]
+    fmt = '  '.join(f'{{:{a}{w}}}' for w, a in zip(col_w, align + '<' * 99))
```
Comment (gtsystem) [inline remotezip.py:274]: f-strings doesn't work in python 2.7. It's probably time to remove compatibility for it, but that would require a major release that is probably outside the scope of this MR.

File: setup.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
     description='Access zip file content hosted remotely without downloading the full file.',
     long_description=description,
     long_description_content_type="text/markdown",
-    install_requires=["requests", "tabulate"],
+    install_requires=["requests"],
```
Comment (gtsystem) [inline setup.py:17]: can you increase patch version above?

File: remotezip.py (hunk 1 above)
Comment (relikd) [reply remotezip.py]: replaced with format()

File: setup.py (hunk 2 above)
Comment (relikd) [reply setup.py:17]: done

**Comment by gtsystem** [comment]
Merged #25 which includes #23 . Closing this one

**Pull request closed without merging**